samples/zpp/sem/src/main.cpp

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) 2019 Erwin Rol <erwin@erwinrol.com>
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <zpp/thread.hpp>
+#include <zpp/sem.hpp>
+#include <zpp/fmt.hpp>
+
+ZPP_THREAD_TCB_DEFINE(tcb1, 1024);
+ZPP_THREAD_TCB_DEFINE(tcb2, 1024);
+ZPP_THREAD_TCB_DEFINE(tcb3, 1024);
+
+namespace {
+
+void thread_entry(zpp::sem* sem_ptr)
+{
+	using namespace zpp;
+	using namespace std::chrono;
+
+	while (true)
+	{
```

Comment:
The PR needs to be run through `uncrustify` to avoid style problems like this, which may require adding new rules to the configuration.

Reply:
Interestingly the CI didn't complain, or the complain would have come after the complain about the documentation. But anyway, details ;-)

Reply:
CI doesn't check coding style beyond basics.